src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkBase.cs

```diff
@@ -34,13 +36,15 @@ abstract class MonoDroidSdkBase
 		// runtimePath: contains Mono.Android.DebugRuntime-*.apk
 		// binPath:     contains mandroid
 		// mscorlibDir: contains mscorlib.dll
-		public void Initialize (string runtimePath = null, string binPath = null, string bclPath = null)
+		public void Initialize (string expectedRuntimePath = null, string binPath = null, string bclPath = null)
```

Comment:
Why rename this parameter? If it is a good idea, then the comment ~3 lines above should also be corrected.